services/ingestion/etl/inforegio/json/src/lib/transform.js

```diff
@@ -8,7 +8,7 @@ import type { Project } from '../../../../types/Project';
 
 const getFundingArea = record =>
   // Get value for 'Funding area' if property is present.
-  (record.Funds ? record.Funds.split(';') : []).filter(
+  (record.Funds ? record.Funds.split(';').filter(fund => fund) : []).filter(
```

Comment:
The current filter of `item => item` can go now that the `.filter(fund => fund)` is here I think.

Reply:
My bad, thanks for noticing